### PR TITLE
rec: stuctured logging for various subsystems

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/nod.cc [hunk 1]
```diff
@@ -98,13 +101,15 @@ bool PersistentSBF::init(bool ignore_pid) {
...
-            g_log<<Logger::Warning<<"NODDB init: Cannot parse file: " << filename << ": " << e.what() << "; removed" << endl;
+            SLOG(g_log<<Logger::Warning<<"NODDB init: Cannot parse file: " << filename << ": " << e.what() << "; removed" << endl,
+                 log->error(Logr::Warning, e.what(), "NODDB init: Cannot parse file, removed", "file",  Logging::Loggable(filename)));
           }
         }
       }
     }
     catch (const filesystem::filesystem_error& e) {
-      g_log<<Logger::Warning<<"NODDB init failed:: " << e.what() << endl;
+      SLOG(g_log<<Logger::Warning<<"NODDB init failed:: " << e.what() << endl,
```
Comment (fredmorcos) [inline pdns/nod.cc:111]: I believe one `:` is enough?

File: pdns/unix_utility.cc [hunk 2]
```diff
@@ -83,23 +84,35 @@ void Utility::setBindAny(int af, sock_t sock)
...
+    }
 #endif
 #ifdef IPV6_BINDANY
-  if (af == AF_INET6)
-    if (setsockopt(sock, IPPROTO_IPV6, IPV6_BINDANY, &one, sizeof(one)) < 0)
-      g_log<<Logger::Warning<<"Warning: IPV6_BINDANY setsockopt failed: "<<stringerror()<<endl;
+  if (af == AF_INET6) {
+    if (setsockopt(sock, IPPROTO_IPV6, IPV6_BINDANY, &one, sizeof(one)) < 0) {
+      SLOG(g_log<<Logger::Warning<<"Warning: IPV6_BINDANY setsockopt failed: "<<stringerror(err)<<endl,
+           g_slog->withName("runtime")->error(Logr::Warning, err, ""Warning: IPV6_BINDANY setsockopt failed"));
```
Comment (fredmorcos) [inline pdns/unix_utility.cc:106]: Umm.. what's going on with the two double-quotes here? `""Warning: ...`

**Review by fredmorcos — APPROVED** [review]
Minor stuff, nothing blocking.

File: pdns/unix_utility.cc (hunk 2 above)
Comment (omoerbeek) [reply pdns/unix_utility.cc:106]: Editing mistake, missed by compiler because of the #ifdef. Thanks for spotting!

**Pull request merged**